packages/openspp/src/Adaptor.js

```diff
@@ -660,6 +630,294 @@ export function unenroll(registrant_id, program_id) {
+          [['name', '=', role]]
+        );
+        if (roleId.length === 0) {
+          roleId = [await sppConnector.create(
+            'g2p.group.membership.kind',
+            { name: role }
+          )];
+        }
+      }
+      const res = await sppConnector.searchRead(
+        'g2p.group.membership',
+        [
+          ['group.registrant_id', '=', group_id],
+          ['individual.registrant_id', '=', individual_id],
+          ['is_ended', '=', false],
+        ],
+        ['id', 'kind'],
+        { limit: 1 },
+      );
+      if (res.length === 0) {
+        const individual = await sppConnector.searchRead(
+          'res.partner',
+          [
+            ['registrant_id', '=', individual_id],
+            ['is_registrant', '=', true],
+            ['is_group', '=', false],
+          ],
+          ['id'],
+          { limit: 1 },
+        );
+        const group = await sppConnector.searchRead(
+          'res.partner',
+          [
+            ['registrant_id', '=', group_id],
+            ['is_registrant', '=', true],
+            ['is_group', '=', true],
+          ],
+          ['id'],
+          { limit: 1 },
+        );
+        if (individual.length === 0 || group.length === 0) {
+          throw new Error(`Individual or Group does not exist!`);
+        }
+        await sppConnector.create(
+          'g2p.group.membership',
+          {
+            individual: individual[0].id,
+            group: group[0].id,
+            kind: [[6, 0, roleId]],
+          }
+        );
+      } else {
+        const groupMembershipIds = res.map( i => i.id );
+        await sppConnector.update(
+          'g2p.group.membership',
+          groupMembershipIds,
+          { kind: [[6, 0, roleId]] },
+        );
+      }
+      if (role.length > 0) {
+        console.log(`ℹ Individual ${individual_id} added to group ${group_id} with role ${role}!`);
+      } else {
+        console.log(`ℹ Individual ${individual_id} added to group ${group_id}!`);
+      }
+    } catch (err) {
+      console.log(`✗ Error: ${err}`);
+    } finally {
+      return state;
+    }
+  }
+}
+
+/**
+ * remove individual from group in OpenSPP
+ * @public
+ * @example
+ * removeFromGroup("GRP_B2BRHJN2", "IND_8DUQL4M4")
+ * @function
+ * @param {string} group_id - group registrant id
+ * @param {string} individual_id - individual registrant id
+ * @returns {Operation}
+ */
+export function removeFromGroup(group_id, individual_id) {
+  return async state => {
+    try {
+      const res = await sppConnector.searchRead(
+        'g2p.group.membership',
+        [
+          ['group.registrant_id', '=', group_id],
+          ['individual.registrant_id', '=', individual_id],
+          ['is_ended', '=', false],
+        ],
+        ['id'],
+      );
+      if (res.length > 0) {
+        const groupMembershipIds = res.map( i => i.id );
+        const now = new Date();
+        const sppDateTimeNowString = dateFns.format(now, 'y-M-d H:m:s');
```

Comment:
```suggestion
        const sppDateTimeNowString = dateFns.format(now, 'y-M-d HH:mm:ss');
```
@josephjclark can you accept this change before merging this into your source? I tested this before but since the time is not right, it did not expose this error